src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -4,13 +4,14 @@
 
 // @flow
 import React, { PureComponent } from "react";
-import * as I from "immutable";
-
 import { connect } from "react-redux";
-import { createSelector } from "reselect";
 import { bindActionCreators } from "redux";
-import { features } from "../../utils/prefs";
+
 import classnames from "classnames";
+import * as I from "immutable";
```

Comment:
Having this in alphabetical order for each of the parts (dependencies, imports) would be great :)